package.json

```diff
@@ -34,6 +34,7 @@
     "clean:build": "rimraf build",
     "lint": "eslint . --cache && echo \"eslint: no lint errors\"",
     "prebuild": "npm run clean:build",
+    "size": "size-limit 115KB build/index.js",
```

Comment:
Now we can say that the jss overhead is minimal with [5%](https://github.com/cssinjs/jss/blob/master/package.json#L105) cc @kof